applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -52,6 +52,9 @@ KratosStructuralMechanicsApplication::KratosStructuralMechanicsApplication():
     mShellThinCorotationalElement3D3N( 0, Element::GeometryType::Pointer( new Triangle3D3 <Node<3> >( Element::GeometryType::PointsArrayType( 3 ) ) ), true ),
     // Adding the membrane element
     mMembraneElement3D3N( 0, Element::GeometryType::Pointer( new Triangle3D3 <Node<3> >( Element::GeometryType::PointsArrayType( 3 ) ) ) ),
+	mMembraneElement3D4N( 0, Element::GeometryType::Pointer( new Quadrilateral3D4 <Node<3> >( Element::GeometryType::PointsArrayType( 4 ) ) ), false ),	
```

Comment:
Exactly, that is why we are staring to use the AD to implement elements, because that way the only thing to "debug" is the Galerkin functional. Besides, in that case I recommend you to chose an alternative name, like PreStressMembrane, or something different. Besides, I thought that the way to introduce prestresses was using a constitutive law, right @RiccardoRossi ?